packages/block-editor/src/components/block-preview/index.js

```diff
@@ -92,6 +93,20 @@ export function BlockPreview( {
  */
 export default memo( BlockPreview );
 
+function InnerStyles() {
+	const { styles } = useSelect( ( select ) => {
```

Comment:
Isn't `styles` empty here since we're passing an empty styles config to the provider?

Reply:
Nope! These `styles` are the styles that are local to the preview, so they're the "real" ones that we wish to output. They've been cleared out of the parent styles, so now have only those styles generated within this local-to-the-preview block editor store.